src/Search/Search.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+import './Search.css';
+import searchIcon from '../images/searchIcon.svg';
+
+const Search = () => {
+    const [ searchInput, setSearchInput ] = useState('');
+    const handleSubmission = (event: any) => {
```

Comment:
For typescript, you generally want to avoid using any as a type.

This what I found on SO, for a better typing for events. (React.ChangeEvent<HTMLInputElement>)

https://stackoverflow.com/questions/40676343/typescript-input-onchange-event-target-value

Reply:
Tried removing the any a few different ways and was getting errors on the varieties I found from that SO article. I still come upon an error on event.preventDefault() where TS is saying that the propety does not exist. How have you handled this in the past?

Reply:
I think "event" might be deprecated, so see if you can get it to work using "e".